src/core/pyproxy.js

```diff
@@ -274,7 +274,7 @@ Module.callPyObject = function (ptrobj, ...jsargs) {
  * @typedef {(PyProxyClass & {[x : string] : Py2JsResult})} PyProxy
  * @typedef { PyProxy | number | bigint | string | boolean | undefined } Py2JsResult
  */
-class PyProxyClass {
+export class PyProxyClass {
```

Comment:
Are these `export` keywords relevant?

Reply:
They were for me: the reason being that these get referenced in the types, so a package importing something else from this file might return this type indirectly. Typescript would complain without it